core/domain/feedback_services.py

```diff
@@ -58,10 +63,21 @@ def create_thread(
     # made there as well
     thread.status = feedback_models.STATUS_CHOICES_OPEN
     thread.subject = subject
+    if has_suggestion:
```

Comment:
Could just do the following, I think?

     thread.has_suggestion = has_suggestion